README.md

```diff
@@ -1 +1,69 @@
-# alp
\ No newline at end of file
+# Aunt Leah's Place Volunteer Management Web App
+This application was built for [Aunt Leah's Place](https://auntleahs.org/) by the [Code The Change Foundation](http://codethechange.ca/). See more of CTC's work [here](https://github.com/CodetheChangeFoundation).
+
+## About
+This application will help Aunt Leah's Place to better manage their volunteers. The application enables volunteers to:
+* create an account
+* sign in for shifts at various events and locations
+It also enables administrators to: 
+* add and delete volunteer locations
+* view and export volunteer data for integration with Salesforce
+* clear volunteer data
+
+## How it was made
+This application was built using React, Redux, Material-UI, and Node.js. The application also uses Azure Functions, an MS SQL database, and Azure's Active Directory for authentication.
+
+### Tech Stack
+* React
+* Redux
+* Material-UI
+* Node.js
+* Azure Functions
+* Azure Active Directory
+* MS SQL
+
+## Try the app locally
+
+### Requirements
+* NPM and [Node.js](https://nodejs.org/en/)
+* Docker
+* Azure Data Studio
+* Azure Functions Extension for Visual Studio Code
+
+### Initial configuration
+1. Clone the repository and open the project in your IDE. We recommend [Visual Studio Code](https://code.visualstudio.com/) to make using Azure a whole lot easier.
+2. If you do opt to use VSCode, install the [Azure Functions](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azurefunctions) extension. [Docker](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-docker) will also be handy, although it's not totally necessary.
+3. In a terminal, navigate to the `aunt-leahs-app` directory and run `npm -i`. Then, open a second terminal window, navigate to  the `aunt-leahs-functions` directory, and do the same. 
+4. In `aunt-leahs-app` run `npm start`. Your browser should open to the app, though it'll crash because there's no server running! 
```

Comment:
```suggestion
4. In the `aunt-leahs-app` directory, run `npm start`. Your browser should open to the app and crash, because there's no server running yet! 
```